### DISCUSSION TITLE
TP4 - Ejercicio 8 - Busqueda

### DISCUSSION THREAD

**Discussion by agusscarava** [INGCOM-UNRN-P1/cursada-2023#142] [Preguntas y respuestas] [answered]
Consigna: 
Implementar una función que retorne el puntero relativo a una cadena segura, en donde se encuentre la primera ocurrencia de una segunda.

Retornen `NULL` si no se encuentra.

```c
char* busqueda(char cadena[], int capacidad_cadena, char busqueda[], int capacidad_busqueda);
```

Hola!
En este ejercicio no entiendo que es lo que tiene que buscar la funcion en la cadena, a que se refiere con " la primera ocurrencia de una segunda." ?

**Comment by martinvilu** [comment]
Cadena: `Hola Mundo Programación`
Búsqueda: `Mundo`
Puntero a la `M` en cadena

**Reply by martinvilu** [reply, marked as answer]
En un ejemplo un poco mas complejo,
Cadena: `El perro marron come carbon en una tarde de con coscorron`
Búsqueda: `on`
Retorno, el primer `on` en la cadena, en la 13 ava posición

**Comment by martinvilu** [comment]
Duplicado (o casi) de https://github.com/INGCOM-UNRN-P1/cursada-2023/discussions/138
